docs/building.md

```diff
@@ -0,0 +1,22 @@
+# Building Atmosphère
+
+The process for building Atmosphère is similar to building Fusée Gelée payloads and other Switch apps.
+
+## Linux
+
+### Preparation
++ If you do not currently have DevkitA64 installed, you can install it [here](https://github.com/devkitPro/pacman/releases/tag/devkitpro-pacman-1.0.1).
```

Comment:
The 'd' is not capitalized. It's devkitA64, devkitARM, devkitPro. You should also link to the official [Getting Started](https://devkitpro.org/wiki/Getting_Started) and [devkitPro pacman](https://devkitpro.org/wiki/devkitPro_pacman) pages, that contain instructions for all OSes including several different flavors of Linux.

Reply:
Ok I will fix that tonight

Reply:
@fincs Anything else that you think might need to be changed/improved upon? We're always open to constructive feedback.